shortly/app/main.py

```diff
@@ -1,3 +1,9 @@
 from fastapi import FastAPI
 
+from shortly.app.events import setup_events
+
 app = FastAPI()
+
+# setup FastAPI components
```

Comment:
I don't think we need this comment here 🤔

Reply:
After thinking about it, you're right. 
Function name setup_* itself indicates installations.

Reply:
Done